garage/theano/envs/base.py

```diff
@@ -22,8 +22,8 @@ class TheanoEnv(GarageEnv):
         env (gym.Env): the env that will be wrapped
     """
 
-    def __init__(self, env):
-        super(TheanoEnv, self).__init__(env)
+    def __init__(self, env=None, env_name=""):
+        super(TheanoEnv, self).__init__(env, env_name)
```

Comment:
Should it be `super().__init__(env, env_name)`?

Reply:
super() will call the constructor of the *first* parent class only. If your class uses mixins/multiple inheritance, you have to explicitly call the test of the parent constructors if you need them.